src/components/apps/savedLaunch/SavedLaunchListing.js

```diff
@@ -218,50 +252,68 @@ function ListSavedLaunches(props) {
         }
     );
 
-    const embedCodeClickHandler = () => {
-        const shareUrl = getShareUrl(selected.id);
+    const { mutate: createInstantLaunch } = useMutation(addInstantLaunch, {
+        onSuccess: (createdIL, { onSuccess }) => {
+            queryClient.invalidateQueries(ALL_INSTANT_LAUNCHES_KEY);
+            setSelectedIL(createdIL);
+            setUseInstantLaunch(true);
+        },
+        onError: setCreateILError,
+    });
+
+    const getShareUrl = useCallback(() => {
+        const host = getHost();
+        const url = `${host}${getSavedLaunchPath(
+            systemId,
+            appId,
+            selected?.id
+        )}`;
+        return url;
+    }, [selected, systemId, appId]);
+
+    useEffect(() => {
+        var shareUrl = "";
```

Comment:
This can be a [let](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/let):
```suggestion
        let shareUrl = "";
```

Reply:
of course, `let`! I knew there was a better option than `var` but I knew this couldn't be a `const`. Shoulda googled it, heh.